R/modify-model-field.R

```diff
@@ -19,16 +19,28 @@
 #' @param .field Character scalar of the name of the component to modify
 #' @param .value Whatever is to be added to `.mod[[.field]]`, typically a character vector
 #' @param .append If `TRUE`, the default, concatenate new values with currently present values. If `FALSE`, new values will overwrite old values.
+#' @param .remove If `TRUE`, `.value` with be removed from the `.field` instead of added. `FALSE` by default. Cannot have both `.append` and `.remove` be true in the same call.
 #' @param .unique If `TRUE`, the default, de-duplicate `.mod[[.field]]` after adding new values. If `FALSE` duplicate values will be kept.
 #' @export
-modify_model_field <- function(.mod, .field, .value, .append = TRUE, .unique = TRUE) {
+modify_model_field <- function(.mod, .field, .value, .append = TRUE, .remove = FALSE, .unique = TRUE) {
```

Comment:
Feels like this needs `@keywords internal` in the documentation.  Or don't export it?

Reply:
yeah, we export it now, and warn people not to use it directly. I don't know, maybe it should be internal, but I think it's fine at this point.

Reply:
`@keywords internal` would still keep it exported, but drop it from the available functions list.  Might be good compromise if you really don't want people to use it.